src/main/kotlin/me/ezra_home/retail_software_solution/business/util/mapper/userinfo/FruitMapper.kt

```diff
@@ -0,0 +1,38 @@
+package me.ezra_home.retail_software_solution.business.mapper
+
+import org.springframework.stereotype.Component
+import me.ezra_home.retail_software_solution.model.entity.FruitEntity
+import me.ezra_home.retail_software_solution.model.entity.FruitEntityDto
+import java.util.UUID
+
+@Component
+class FruitMapper {
```

Comment:
If you try to write all fields manually, what happens when you have 50 fields. That will be a very long and repetitive file. That is why we use Mapstruct. There is an example for that.